lib/cloudregister/registerutils.py

```diff
@@ -160,8 +160,15 @@ def credentials_files_are_equal(repo_credentials):
     """Compare the base credentials files the the repo header and make
        sure they have the same values."""
     credentials_location = '/etc/zypp/credentials.d/'
+    try:
+        credentials_header = os.path.join(
+            credentials_location,
+            repo_credentials
+        )
+    except TypeError:
+        return False
+
```

Comment:
No `try-except` block this is a simple if condition
```
if not repo_credentials:
    return False
```
Receiving an empty velu for an argument is not an exceptional condition.